webpack.config.js

```diff
@@ -0,0 +1,15 @@
+const { nodeEnv } = require('./config');
+
+let config;
+
+try {
+	// eslint-disable-next-line global-require
+	config = require(`./webpack/${nodeEnv}.config`);
+} catch (error) {
+	console.error(error);
+	console.error(`Couldn't find webpack config for ${nodeEnv} environment, using base config as a fallback\n`);
```

Comment:
Do we really need this? We can leave development config as the default one.

Reply:
Yep, it's needed because it indicates that something went wrong. I'll make production config as a default one because by default development server is not needed so development config doesn't suit well here.

Reply:
done in bd17768741d2c5d966b7e3190b680778a3392697